web/package.json

```diff
@@ -23,6 +23,7 @@
     "prepare": "svelte-kit sync"
   },
   "devDependencies": {
+    "@ernane/svelte-star-rating": "^1.1.7",
```

Comment:
I don't think we should need a whole separate dependency to show a few stars.

Reply:
Ok, I understand. The star library is really very small, I could easily include it in the code as a component, if that makes sense?

Reply:
I'll let Alex have the final call on the looks, but to me the styling of this star-rating component looks pretty out of place in Immich, so it might be better to just make our own thing.